### PR TITLE
List billing projects

### LINKED ISSUE #1182: List billing projects
As a geneticist performing analyses, I want to view a list of the projects I have access to, in order to confirm my access and permission level

### PR DESCRIPTION
**Pull request by elizzhao**
Fixes #1182 

Used a table per Lou's suggestion and showed it to Jerome. The long cards from groups looked odd as billing doesn't have as much information to fill up the card as the groups do. 

EDIT: am unskilled with git so all of isaac's changes from merging his branch into mine showed up as changes here too, oops. All files of interest for this PR are:

- TopBar (adding the billing button)
- Router (adding the billing projects list page)
- BillingProjectsList.js file (all new, most of the changes!)

### CODE REVIEW

**Comment by elizzhao** [comment]
Oh shoot--didn't realize merging Isaac's changes into mine would show us as commits/changes too...

Things of concern are adding the button to the side-nav in Top Bar and a new BillingProjectList page

**Review by zarsky-broad — APPROVED** [review]

File: src/pages/BillingProjectsList.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+                  grow: 0
+                },
+                headerRenderer: () => h(HeaderCell, ['Role']),
+                cellRenderer: ({ rowIndex }) => { return billingProjects[rowIndex].role }
+              }
+            ]
+          })
+        ]),
+        !isDataLoaded && spinnerOverlay,
+        updating && spinnerOverlay
```
Comment (zarsky-broad) [inline src/pages/BillingProjectsList.js]:
```js
(!isDataLoaded || updating) && spinnerOverlay
```

File: src/pages/BillingProjectsList.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+                headerRenderer: () => h(HeaderCell, ['Project Name']),
+                cellRenderer: ({ rowIndex }) => { return billingProjects[rowIndex].projectName }
+              },
+              {
+                size: {
+                  basis: 150,
+                  grow: 0
+                },
+                headerRenderer: () => h(HeaderCell, ['Role']),
+                cellRenderer: ({ rowIndex }) => { return billingProjects[rowIndex].role }
```
Comment (zarsky-broad) [inline src/pages/BillingProjectsList.js]:
```suggestion
                cellRenderer: ({ rowIndex }) => billingProjects[rowIndex].role
```

File: src/pages/BillingProjectsList.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+                      }
+                    }),
+                    billingProjects[rowIndex].creationStatus
+                  ])
+                }
+              },
+              {
+                size: { basis: 200 },
+                headerRenderer: () => h(HeaderCell, ['Project Name']),
+                cellRenderer: ({ rowIndex }) => { return billingProjects[rowIndex].projectName }
```
Comment (zarsky-broad) [inline src/pages/BillingProjectsList.js]:
```suggestion
                cellRenderer: ({ rowIndex }) => billingProjects[rowIndex].projectName
```

File: src/pages/BillingProjectsList.js [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+            rowCount: billingProjects.length,
+            columns: [
+              {
+                size: {
+                  basis: 120,
+                  grow: 0
+                },
+                headerRenderer: () => h(HeaderCell, ['Status']),
+                cellRenderer: ({ rowIndex }) => {
+                  return h(Fragment, [
```
Comment (zarsky-broad) [inline src/pages/BillingProjectsList.js:93]:
```js
const projectReady = billingProjects[rowIndex].creationStatus === 'Ready'
```

**Review by zarsky-broad — COMMENTED** [review]
Do you use `updating` at all?

**Comment by elizzhao** [comment]
I don't think it actually does anything different from isDataLoaded--I'll get rid of it. I think it might be useful later when we create/delete billing projects, but for now it isn't serving any purpose. Thanks for the catch!

**Comment by katevoss** [comment]
You should definitely notify Comms & @bradtaylor that this is going out today, in 40 minutes!

**Pull request merged**